### PR TITLE
Update FDC template

### PR DESCRIPTION
**Pull request by maneesht**

### CODE REVIEW

File: data-connect/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Firebase Data Connect
+
+<a href="https://idx.google.com/new?template=https://github.com/project-idx/community-templates/tree/mtewani-flutter-template/dataconnect">
```
Comment (rodydavis) [inline data-connect/README.md:3]: Needs to be /templates and also not your branch

File: data-connect/flutter-blank/dev.nix [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# To learn more about how to use Nix to configure your environment
+# see: https://developers.google.com/idx/guides/customize-idx-env
+{ pkgs, ... }: {
+  processes = {
+    writeEnv = {
+      command = "echo \"HOST=$WEB_HOST\" > .env";
```
Comment (rodydavis) [inline data-connect/flutter-blank/dev.nix:6]: This should be added to env {}, and also inside the preview env args

File: data-connect/flutter-blank/lib/generated/default.dart [hunk 3]
```diff
@@ -0,0 +1,30 @@
+library default_connector;
+import 'package:firebase_data_connect/firebase_data_connect.dart';
+import 'dart:convert';
+
+
```
Comment (rodydavis) [inline data-connect/flutter-blank/lib/generated/default.dart:5]: Is there auto formatting that can be done for these files?

File: data-connect/flutter-blank/proxy-web/proxy.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+const express = require("express");
+const { createProxyMiddleware } = require("http-proxy-middleware");
+
+const EXT_PORT = 9002;
```
Comment (rodydavis) [inline data-connect/flutter-blank/proxy-web/proxy.js:4]: Can these look for a ENV and fallback to these values if not set? To help with updating in the future

File: data-connect/flutter-blank/proxy-web/proxy.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    router: {
+      [`/v1beta`]: `http://localhost:${DATACONNECT_PORT}`, // API on 8081 should support /api/* paths
+    },
+  })
+);
+
+app.listen(EXT_PORT, () => {
+  console.log(`
+    ====Firebase Data Connect Proxy Server========
+    ====Please DO NOT CLOSE=======================
```
Comment (rodydavis) [inline data-connect/flutter-blank/proxy-web/proxy.js:21]: You can also start this in [nohup](https://en.wikipedia.org/wiki/Nohup)

File: data-connect/flutter-movie/lib/movies_connector/add_review.dart [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+    Map<String, dynamic> json = {};
+    
+      
+      json['review_upsert'] = 
+  
+      review_upsert.toJson()
+  
+;
+      
+
```
Comment (rodydavis) [inline data-connect/flutter-movie/lib/movies_connector/add_review.dart:129]: Can these files be formatted?

**Review by rodydavis — CHANGES_REQUESTED** [review]

File: data-connect/README.md (hunk 1 above)
Comment (maneesht) [reply data-connect/README.md]: Updated.

**Review by rodydavis — APPROVED** [review]
LGTM

**Pull request merged**